CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Go-Common is the library of common functions for Tidepool's Go-based applications
 
+## 1.1.0 - 2021-08-19
+### Changed
+- YLP-956: migrate log to lorgus libraries
```

Comment:
Why?

Reply:
i added the explanation in the JIRA ticket and inside this PR